### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by haliljon**
## In this PR I was able to:

> - use Javascript syntaxes.
> - use JavaScript to manipulate DOM elements.
> - use JavaScript events.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .eslintrc.json:1]: - [ ] This is a pair-programming project but i cannot see any commit from your partner's side. Kindly ask your partner to make some contributions in this pull request :blush:

Comment (HAMMAS-SALEEM) [inline .eslintrc.json:1]: - [ ] I cannot see any .gitignore file in your project. Kindly add that file to follow the requirements :blush:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Pull Request :red_circle:

## Hey Team :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the reqirements, But i cannot proceed review on your project there are some issues that you still need to work on :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @haliljon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**